src/PaymentGateways/PayPalCommerce/ScriptLoader.php

```diff
@@ -151,12 +151,15 @@ public function loadPublicAssets()
             'components' => 'hosted-fields,buttons',
             'locale' => get_locale(),
             'disable-funding' => 'credit',
-            'enable-funding' => give_is_setting_enabled( $settings['paypal_commerce_accept_venmo'] ) ? 'venmo' : '',
             'vault' => true,
             'data-partner-attribution-id' => give('PAYPAL_COMMERCE_ATTRIBUTION_ID'),
             'data-client-token' => $this->merchantRepository->getClientToken(),
         ];
 
+        if (give_is_setting_enabled($settings['paypal_commerce_accept_venmo'])) {
```

Comment:
@jonwaldstein I am getting PHP notice on the existing install. I updated logic to use `give_get_option` function to prevent PHP notice: https://github.com/impress-org/givewp/compare/475f28c05c35...47e19c464abb

Reply:
@ravinderk what notice did you encounter and did you test this?  your edit makes it so it will always return true because you have a default string of 'disabled'.

Reply:
@jonwaldstein Good catch. I fixed it:https://github.com/impress-org/givewp/compare/47e19c464abb...83f1000ee915

I was setting the following error in `develop` branch.
```
PHP Warning:  Undefined array key "paypal_commerce_accept_venmo" in /Users/ravinderkumar/ValetSites/givewp/wp-content/plugins/give/src/PaymentGateways/PayPalCommerce/ScriptLoader.php on line 159```